omnigibson/scenes/scene_base.py

```diff
@@ -259,13 +296,17 @@ def _load(self):
     def _load_systems(self):
         system_dir = os.path.join(gm.DATASET_PATH, "systems")
 
+        available_systems = dict()
         if os.path.exists(system_dir):
             system_names = os.listdir(system_dir)
             for system_name in system_names:
```

Comment:
maybe you intend to use `get_all_system_names` here?

Reply:
done